yazi-boot/src/actions/debug.rs

```diff
@@ -79,17 +82,22 @@ impl Actions {
 			"    file             : {}",
 			Self::process_output(env::var_os("YAZI_FILE_ONE").unwrap_or("file".into()), "--version")
 		)?;
-		writeln!(s, "    ueberzugpp       : {}", Self::process_output("ueberzugpp", "--version"))?;
-		writeln!(s, "    ffmpegthumbnailer: {}", Self::process_output("ffmpegthumbnailer", "-v"))?;
-		writeln!(s, "    magick           : {}", Self::process_output("magick", "--version"))?;
-		writeln!(s, "    fzf              : {}", Self::process_output("fzf", "--version"))?;
-		writeln!(s, "    fd               : {}", Self::process_output("fd", "--version"))?;
-		writeln!(s, "    rg               : {}", Self::process_output("rg", "--version"))?;
-		writeln!(s, "    chafa            : {}", Self::process_output("chafa", "--version"))?;
-		writeln!(s, "    zoxide           : {}", Self::process_output("zoxide", "--version"))?;
-		writeln!(s, "    7z               : {}", Self::process_output("7z", "i"))?;
-		writeln!(s, "    7zz              : {}", Self::process_output("7zz", "i"))?;
-		writeln!(s, "    jq               : {}", Self::process_output("jq", "--version"))?;
+
+		for (cmd, arg) in &[
+			("ueberzugpp", "--version"),
+			("ffmpegthumbnailer", "-v"),
+			("magick", "--version"),
+			("fzf", "--version"),
+			("fd", "--version"),
+			("rg", "--version"),
+			("chafa", "--version"),
+			("zoxide", "--version"),
+			("7z", "i"),
+			("7zz", "i"),
+			("jq", "--version"),
+		] {
+			writeln!(s, "    {:17}: {}", cmd, Self::process_output(cmd, arg))?;
```

Comment:
This doesn't seem to make maintenance easier, since now I have to manually figure out which one is the longest and calculate its length. 

The previous code was WYSIWYG — if it looked aligned in the editor, the output would be aligned too, and I could freely fill in the missing spaces. 

What's worse is that for things like `file`, which need special logic, I still have to handle them the old way — manually filling in the spaces. This creates two different styles.